src/arkode/arkode_erkstep.c

```diff
@@ -766,6 +766,18 @@ int erkStep_TakeStep(void* arkode_mem, realtype *dsmPtr, int *nflagPtr)
                      ark_mem->nst, ark_mem->h, ark_mem->tcur);
 #endif
 
+  /* if fullrhs has not been called, fill in F[0] */
+  if (!ark_mem->call_fullrhs_start) {
```

Comment:
It seems like it may be safer to handle this even though it is not _currently_ possible to reach it.  If its a broke state to reach this, then an error should be thrown if this case is reached.